### ISSUE TITLE
[Bug] CaptureOutput in bilib  (bisync) restores wrong output for logrus 

### ISSUE THREAD

**Issue by dim0x69** [rclone/rclone#8158]
Regarding CaptureOutput in bilib:

https://github.com/rclone/rclone/blob/b9207e57274cd6e7c488c5bd751a058ba0f8b3b9/cmd/bisync/bilib/output.go#L13

First in logrusSave the InfoLevel-Writer ist stored:

https://github.com/rclone/rclone/blob/b9207e57274cd6e7c488c5bd751a058ba0f8b3b9/cmd/bisync/bilib/output.go#L15

... which is then later restored as the default output destination of the logger:

https://github.com/rclone/rclone/blob/b9207e57274cd6e7c488c5bd751a058ba0f8b3b9/cmd/bisync/bilib/output.go#L27

I think what should be stored instead is the Output of the StandardLogger:

logrusSave := logrus.StandardLogger().Out

Currently, if rclone is used as a library and logrus is used after a call to rc "sync/bisync", logging does not work anymore. It leads to writing to a closed pipe. For example, this leads to a write to a closed pipe:

```go
import (
	logger "github.com/sirupsen/logrus"
)
...


out, status := librclone.RPC("sync/bisync", string(bisyncConfigJson))

if status != 200 {
  logger.Fatal(fmt.Sprintf("sync/bisync failed: %d\n%s", status, out))
}
````

**Comment by nielash** [comment]
Thanks for the report! I've commented on the PR.

**Comment by ncw** [comment]
The fix for this is merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.69 

Thank you @dim0x69
